src/v3/src/util/theme.test.ts

```diff
@@ -112,116 +124,38 @@ describe('theme utilities', () => {
       expect(theme.palette?.success).toEqual(palette);
     });
 
-    describe('should apply design tokens -> theme properties', () => {
-      test.each([
```

Comment:
Yeah these tests were there to test the behavior of the deleted lines that you called out here: https://github.com/okta/okta-signin-widget/pull/3471#discussion_r1408282201

Since we deleted this behavior of overriding `theme` properties with our design tokens in favor of injecting the token overrides into the theme creation, these tests weren't necessary anymore